modules/apps/account/account-test/src/testFunctional/macros/AccountUsers.macro

```diff
@@ -5,6 +5,11 @@ definition {
 
 		Pause(locator1 = "3000");
 
+		LiferayEvent.waitForLiferayEvent(
+			attributeName = "src",
+			attributeValue = "dialog-iframe-popup",
```

Comment:
This, at some point is doing:

```
event.src.indexOf('dialog-iframe-popup') !== -1
```

We have urls of the like:

```
.../PAGE?p_p_id="FOO"&className="dialog-iframe-popup"
.../OTHER_PAGE?p_p_id="BAR"&className="dialog-iframe-popup"
```

The original goal of the `attributeName`/`attributeValue` pair was to be able to filter unrelated occurrences of events. If you want a partial match, you should still pick the parts of the URL that are unique to your occurrence. Best thing would be to use as close a match as possible to avoid indeterminism.

@kenjiheigel, this is the risk of trying to be too clever with APIs. They're often misused and induce errors rather than help.